Logger.go

```diff
@@ -8,10 +8,8 @@ import (
     "sync"
 )
 
-// Logger ...
-type Logger struct {
-    logEntry *logrus.Entry
-    logLevel logrus.Level
+type logger struct {
+    *logrus.Logger
```

Comment:
Nice, nooit over nagedacht dat een struct met een kleine letter private is... zo kan je het gebruik van de constructor-methode dus afdwingen.